registration/tests/serializers/test_family.py

```diff
@@ -23,41 +28,89 @@ def setUp(self):
             address="1 Django Boulevard",
             preferred_comms="Shark Tune",
         )
-
         self.family_without_children = Family.objects.create(
             parent=self.parent1,
             email="justkeepswimming@ocean.com",
             cell_number="123456789",
             address="42 Wallaby Way",
             preferred_comms="Whale Song",
         )
-
-        self.family_without_parent = Family.objects.create(
+        self.family_status = Family.objects.create(
```

Comment:
nit: `family_with_enrolment` might be a bit clearer

would also be great to rename `parent3` to express its purpose – something like `parent_with_enrolment` could work?